feature-odyssey/src/main/kotlin/com/hedvig/android/odyssey/search/SearchActivity.kt

```diff
@@ -40,8 +48,10 @@ class SearchActivity : ComponentActivity() {
 
       val commonClaimId = viewState.selectedClaim?.id
       LaunchedEffect(commonClaimId) {
-        if (commonClaimId != null) {
-          startClaimsFlow(viewModel, commonClaimId)
+        lifecycleScope.launch {
+          if (commonClaimId != null) {
+            startClaimsFlow(viewModel, commonClaimId)
+          }
```

Comment:
I think inside the LaunchedEffect you're in a suspending context already, so you can call startClaimsFlow directly without launching a new coroutine, can you try that and see if it works?